scripts/util.cjs

```diff
@@ -1,10 +1,12 @@
-const childProcess = require('child_process')
-const fs = require('fs')
-const os = require('os')
-const path = require('path')
-const sharp = require('sharp')
-const Downloader = require('nodejs-file-downloader')
-const fetch = require('node-fetch')
+const childProcess = require("child_process")
+const fs = require("fs")
+const os = require("os")
+const path = require("path")
+const sharp = require("sharp")
+const fetch = require("node-fetch")
+const { extension } = require("mime-types")
+const { https } = require("follow-redirects")
+const trustedCoingeckoIds = require('../data/solana/trusted-coingecko-ids.json')
```

Comment:
nit: maybe rename to `trustedCoingeckoIdsByChainId`, that way we know this is an object and not a list of ids.